guide/src/main/asciidoc/problems/badRequest.adoc

```diff
@@ -63,6 +86,14 @@ Content-Type: application/problem+json
       "value": "12345678901",
       "replacedBy": "23456789012"
     },
+    {
+      "type": "urn:problem-type:belgif:input-validation:referencedResourceNotFound",
+      "title": "Referenced resource was not found.",
+      "detail": "A person with SSIN 98765432109 was not found",
```

Comment:
In https://github.com/belgif/rest-problem-java/blob/c36e0c9ab0e410b8fd2846d0fb7bfe520a645af0/belgif-rest-problem/src/main/java/io/github/belgif/rest/problem/api/InputValidationIssues.java#L105 the default detail message is "Referenced resource boardMembers[1].ssin = '98765432109' does not exist". We should align to one or the other.